third_party/kubeadm/README.md

```diff
@@ -0,0 +1,13 @@
+maintaining a fork of kubernetes constants to avoid depending on k8s.io/kubernetes/ as a lib
+how to update this fork
+
+# clone kuberentes
+
+# pick latest stable version
+$ git reset --hard v1.22.4
```

Comment:
probably the fastest way to get only a specific tag instead of cloning the whole k8s is:
```
git clone --depth 1 --branch v1.22.4 git@github.com:kubernetes/kubernetes.git
```

this would download ~30MB of objects vs ~1GB of objects.